### PR TITLE
[Develop] Hot-fix OV inference issue in rotated detection

### PR DESCRIPTION
**Pull request by harimkang**
## Summary
Hot-fix for OV Inference issue of rotated detection
+ black format for openvino_task.py

### CODE REVIEW

**Comment by harimkang** [comment]
run ote-test

File: external/mmdetection/detection_tasks/apis/detection/openvino_task.py [hunk 1]
```diff
@@ -136,36 +154,53 @@ def convert2array(self, detections: List[utils.Detection]) -> np.ndarray:
...
             detections = self.convert2array(detections)
-        if not isinstance(detections, np.ndarray) and isinstance(self, OpenVINODetectionInferencer):
+        if not isinstance(detections, np.ndarray) and isinstance(
+            self, OpenVINODetectionInferencer
+        ):
             detections = np.array(detections)
-        if isinstance(self.converter, MaskToAnnotationConverter):
+        if isinstance(self.converter, MaskToAnnotationConverter) or isinstance(
+            self.converter, RotatedRectToAnnotationConverter
+        ):
```
Comment (goodsong81) [inline external/mmdetection/detection_tasks/apis/detection/openvino_task.py]:
```suggestion
        if isinstance(self.converter, (MaskToAnnotationConverter, RotatedRectToAnnotationConverter)
        ):
```

**Review by goodsong81 — CHANGES_REQUESTED** [review]

File: external/mmdetection/detection_tasks/apis/detection/openvino_task.py (hunk 1 above)
Comment (harimkang) [reply external/mmdetection/detection_tasks/apis/detection/openvino_task.py]: Fixed it!

**Comment by harimkang** [comment]
run ote-test

**Review by goodsong81 — APPROVED** [review]
LGTM. Let's see the test result.

**Pull request merged**